src/management/cameras.php

```diff
@@ -0,0 +1,109 @@
+<?php
+$servername = "db";
+$username = "db_username";
+$password = "db_password";
+$dbname = "db_name";
+
+// Create connection
+$conn = new mysqli($servername, $username, $password, $dbname);
```

Comment:
In this case, we should add `@` sign before `new`, as if the connection fails it will return standard PHP warning. This scenario is already handled below so it shouldn't cause any problems.

```suggestion
$conn = @new mysqli($servername, $username, $password, $dbname);
```